app/res/values/arrays.xml

```diff
@@ -75,5 +75,20 @@
 		<item>Hamlie</item>
 		<item>Nehasie</item>
 		<item>Pagumien</item>
-	</string-array>    
+	</string-array>   
+	<string-array
+	    name="nepali_months">
```

Comment:
Will add one (month names will be copied from here http://en.wikipedia.org/wiki/Nepali_calendar)